src/pretix/static/pretixpresale/js/ui/main.js

```diff
@@ -501,6 +506,80 @@ $(function () {
+        if (!currentTime.isSame(startTime, 'day')) {
+            // Not on same day
+            return;
+        }
+
+        // scroll to best matching tick
+        var currentTimeCmp = parseInt(currentTime.format("Hmm"), 10);
+        var ticks = this.querySelectorAll(".ticks li");
+        var currentTick;
+        var t;
+        for (var i=0, max=ticks.length; i < max; i++) {
+            currentTick = ticks[i]
+            t = parseInt(currentTick.getAttribute("data-start").replace(":", ""), 10);
+            if (t > currentTimeCmp) {
+                break;
+            }
+        }
+        currentTick.scrollIntoView({behavior:"smooth", inline: "center"});
+
+
+        var thisCalendar = this;
+        var currentTimeInterval;
+
+        var timeFormat = document.body.getAttribute("data-timeformat");
+        var timeFormatParts = timeFormat.match(/([a-zA-Z_\s]+)([^a-zA-Z_\s])(.*)/);
+        if (!timeFormatParts) timeFormatParts = [timeFormat];
+        if (timeFormatParts.length > 1) timeFormatParts.shift();
+        var currentTimeBar = $('<div class="current-time-bar" aria-hidden="true"><time></time></div>').appendTo(this);
+        var currentTimeDisplay = currentTimeBar.find("time");
+        var currentTimeDisplayParts = [];
+        timeFormatParts.forEach(function(format) {
+            currentTimeDisplayParts.push([format, $("<span></span>").appendTo(currentTimeDisplay)])
+        }); 
+        var duration = this.getAttribute("data-duration").split(":").reduce(function(previousValue, currentValue, currentIndex) {
+            return previousValue + (currentIndex ? parseInt(currentValue, 10) * 60 : parseInt(currentValue, 10) * 60 * 60);
+        }, 0);
+        function setCurrentTimeBar() {
+            var currentTime = moment().tz(timezone);
+            var currentTimeDelta = Math.floor((currentTime - startTime)/1000);
+            if (currentTimeDelta < 0 || currentTimeDelta > duration) {
+                // Too early || Too late
+                window.clearInterval(currentTimeInterval);
+                currentTimeBar.remove();
+                return;
+            }
+            
+            var offset = thisCalendar.querySelector("h3").getBoundingClientRect().width;
+            var dx = Math.round(offset + (thisCalendar.scrollWidth-offset)*(currentTimeDelta/duration));
+            currentTimeDisplayParts.forEach(function(part) {
+                part[1].text(currentTime.format(part[0]));
+            });
+            if (currentTimeDisplay.get(0).getBoundingClientRect().width + dx >= thisCalendar.scrollWidth) {
+                currentTimeBar.addClass("swap-side");
+            }
+            else {
+                currentTimeBar.removeClass("swap-side");
+            }
+            thisCalendar.style.setProperty('--current-time-offset', dx + "px");
+        }
+        currentTimeInterval = window.setInterval(setCurrentTimeBar, 15*1000);
+        $(window).resize(setCurrentTimeBar);
```

Comment:
I think ``$(window).on("resize", setCurrentTimeBar);`` would make this one think less that breaks if we ever upgrade jquery